.travis/run.sh

```diff
@@ -24,10 +24,14 @@ if [[ $TOXENV != docs ]]; then
     fi
 fi
 
-if [[ $TOXENV == py* ]]; then
+if [[ $TOXENV == py*-functional-install ]]; then
+    # Only run test_install*.py integration tests
+    tox -- -m integration -n 4 --duration=5 --only_install_tests
```

Comment:
Would `-m integration - k "not test_install" work equally well?

Instead of the additional code in conftest and the custom flag...

Reply:
Yes, that's much cleaner, and it also doesn't list hundreds of skipped tests, because they're not selected in the first place.

Updated!